lib/cli/src/commands/init.rs

```diff
@@ -0,0 +1,545 @@
+            )
+            .lines()
+            .collect::<Vec<_>>()
+            .join(NEWLINE);
+
+        std::fs::write(&path, &toml_string)
+            .with_context(|| format!("Unable to write to \"{}\"", path.display()))?;
+
+        Ok(())
+    }
+
+    fn target_file(&self) -> Result<(String, PathBuf), anyhow::Error> {
+        match self.out.as_ref() {
+            None => {
+                let current_dir = std::env::current_dir()?;
+                let package_name = current_dir
+                    .canonicalize()?
+                    .file_stem()
+                    .and_then(|s| s.to_str())
+                    .map(|s| s.to_string())
+                    .ok_or_else(|| anyhow::anyhow!("no current dir name"))?;
+                Ok((package_name, current_dir.join(WASMER_TOML_NAME)))
+            }
+            Some(s) => {
+                let _ = std::fs::create_dir_all(s);
+                let package_name = s
+                    .canonicalize()?
+                    .file_stem()
+                    .and_then(|s| s.to_str())
+                    .map(|s| s.to_string())
+                    .ok_or_else(|| anyhow::anyhow!("no dir name"))?;
+                Ok((package_name, s.join(WASMER_TOML_NAME)))
+            }
+        }
+    }
+
+    fn get_filesystem_mapping(include: &[String]) -> Option<HashMap<String, PathBuf>> {
+        if include.is_empty() {
+            return None;
+        }
+
+        Some(
+            include
+                .iter()
+                .map(|path| {
+                    if path == "." || path == "/" {
+                        return ("/".to_string(), Path::new("/").to_path_buf());
+                    }
+
+                    let key = format!("./{path}");
+                    let value = Path::new(&format!("/{path}")).to_path_buf();
+
+                    (key, value)
+                })
+                .collect(),
+        )
+    }
+
+    fn get_command(
+        modules: &[wapm_toml::Module],
+        bin_or_lib: BinOrLib,
+    ) -> Option<Vec<wapm_toml::Command>> {
+        match bin_or_lib {
+            BinOrLib::Bin => Some(
+                modules
+                    .iter()
+                    .map(|m| {
+                        wapm_toml::Command::V1(wapm_toml::CommandV1 {
+                            name: m.name.clone(),
+                            module: m.name.clone(),
+                            main_args: None,
+                            package: None,
+                        })
+                    })
+                    .collect(),
+            ),
+            BinOrLib::Lib | BinOrLib::Empty => None,
+        }
+    }
+
+    /// Returns the dependencies based on the `--template` flag
+    fn get_dependencies(template: Option<&Template>) -> HashMap<String, String> {
+        let mut map = HashMap::default();
+
+        match template {
+            Some(Template::Js) => {
+                map.insert("python".to_string(), "quickjs/quickjs@latest".to_string());
```

Comment:
I was testing `wasmer init` locally noticed we have a copy/paste bug here.

```toml
[dependencies]
python = 'quickjs/quickjs@latest'
```

```suggestion
                map.insert("quickjs".to_string(), "quickjs/quickjs@latest".to_string());
```